frontend/package.json

```diff
@@ -10,10 +10,10 @@
   "workspaces": {
     "packages": [
       "packages/*"
-    ],
-    "nohoist": [
-      "**"
```

Comment:
`nohoist: **` was removed since we need packages to share the same `node_modules` for things like `react` and `react-dom`